src/dfmt/main.d

```diff
@@ -211,14 +211,15 @@ else
                 else
                     break;
             }
-            format("stdin", buffer, output.lockingTextWriter(), &config);
+            return format("stdin", buffer, output.lockingTextWriter(), &config);
```

Comment:
pretty sure this will have the opposite effect of what is intended: return `true` = error code 1 (failure), return false = error code 0 (success)

Reply:
What's really dumb is that I got this right in the other branch of the code...